safesens/account/schema.py

```diff
@@ -65,10 +65,18 @@ def mutate_and_get_payload(cls, root, info, **input):
 
 class AccountQuery(graphene.ObjectType):
     own_users = DjangoFilterConnectionField(UserNode, filterset_class=UserFilter)
+    get_technicians = DjangoFilterConnectionField(UserNode, filterset_class=UserFilter)
+    ###get all the users that are technicians
```

Comment:
try to clean the code before merging. Only leave comments that explains what you are doing.